### PR TITLE
Adding Townhall Stats 1.10

### PR DESCRIPTION
**Pull request by Asherslab**
* Adding Townhall Stats

Added stats on Info Page of Towhall for miners, fishermen, guards, lumberjacks and farmer. Also moved Total Happiness from Actions page to Info Page

* remove unused import

* Gets rid of Sonars Nested Ternary Operations error

I know it's ugly but i can't really think of another way to do it, sorry, i don't completely understand lambdas yet.

* Making codacy happy

Changed a couple variable to immutable (final)

* Sonar Check

I'd like to see what sonar says about this.

* Moved stuff into TranslationConstats.java

* Un-Nested ternary Operations

I think this one is clear, what do you guys think? also made codacy happy bye changing something to Immutable

* removed unnecessary comparisons

and got confused about sonar. lol. think i got it though

* Moved Open Curly braces

Wow, just wow.

* Moved more curly braces....

* Un-Nested ternary Operations 2.0

Wow, so much simpler

* Got rid of un-used import

Woops

Closes #
Closes #
Closes #

# Changes proposed in this pull request:
- More stats in town hall and other stuff
- Literally the same PR as the one for 1.11
- I honestly haven't tested it for 1.10 because i don't have that version on my mac.

Review please

### CODE REVIEW

**Review by xavierh — APPROVED** [review]

**Comment by Asherslab** [comment]
Should i just always add the "in progress" label?

**Comment by MinecoloniesSonar** [comment]
SonarQube analysis reported 34 issues
* ![MAJOR][MAJOR] 2 major
* ![MINOR][MINOR] 32 minor

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [WindowTownHall.java#L848](https://github.com/Minecolonies/minecolonies/blob/ee86677d3bf5566a8a761fb05f853755610e14c5/src/main/java/com/minecolonies/coremod/client/gui/WindowTownHall.java#L848): Reduce this anonymous class number of lines from 24 to at most 20, or make it a named class. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1188)
1. ![MAJOR][MAJOR] [WindowTownHall.java#L945](https://github.com/Minecolonies/minecolonies/blob/ee86677d3bf5566a8a761fb05f853755610e14c5/src/main/java/com/minecolonies/coremod/client/gui/WindowTownHall.java#L945): Reduce this anonymous class number of lines from 40 to at most 20, or make it a named class. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1188)
1. ![MINOR][MINOR] [blockout](https://github.com/Minecolonies/minecolonies/blob/ee86677d3bf5566a8a761fb05f853755610e14c5/src/main/java/com/minecolonies/blockout): Add a 'package-info.java' file to document the 'blockout' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [controls](https://github.com/Minecolonies/minecolonies/blob/ee86677d3bf5566a8a761fb05f853755610e14c5/src/main/java/com/minecolonies/blockout/controls): Add a 'package-info.java' file to document the 'controls' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [views](https://github.com/Minecolonies/minecolonies/blob/ee86677d3bf5566a8a761fb05f853755610e14c5/src/main/java/com/minecolonies/blockout/views): Add a 'package-info.java' file to document the 'views' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [blocks](https://github.com/Minecolonies/minecolonies/blob/ee86677d3bf5566a8a761fb05f853755610e14c5/src/main/java/com/minecolonies/coremod/blocks): Add a 'package-info.java' file to document the 'blocks' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [gui](https://github.com/Minecolonies/minecolonies/blob/ee86677d3bf5566a8a761fb05f853755610e14c5/src/main/java/com/minecolonies/coremod/client/gui): Add a 'package-info.java' file to document the 'gui' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [WindowTownHall.java#L869](https://github.com/Minecolonies/minecolonies/blob/ee86677d3bf5566a8a761fb05f853755610e14c5/src/main/java/com/minecolonies/coremod/client/gui/WindowTownHall.java#L869): Define the locale to be used in this String operation. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1449)
1. ![MINOR][MINOR] [render](https://github.com/Minecolonies/minecolonies/blob/ee86677d3bf5566a8a761fb05f853755610e14c5/src/main/java/com/minecolonies/coremod/client/render): Add a 'package-info.java' file to document the 'render' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [colony](https://github.com/Minecolonies/minecolonies/blob/ee86677d3bf5566a8a761fb05f853755610e14c5/src/main/java/com/minecolonies/coremod/colony): Add a 'package-info.java' file to document the 'colony' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by xavierh** [comment]
The In progress is et automatically. When I am happy with my prs, I remove that flag , and flag it ready instead.

**Comment by Asherslab** [comment]
okay, cool. I'm happy with mine, so I'm gonna remove it okay?

**Comment by xavierh** [comment]
Yes

File: src/main/resources/assets/minecolonies/lang/en_us.lang [hunk 1]
```diff
@@ -378,27 +383,22 @@ com.minecolonies.coremod.permission.shoot_arrow=Shoot arrows
...
+com.minecolonies.coremod.saturation.5=I'm hungry, I'm not that productive anymore
+com.minecolonies.coremod.saturation.7=I'm quite saturated, but I'll have to eat soon again
 
 com.minecolonies.coremod.item.permissionscepter.setmode=Set to mode '%s'
 com.minecolonies.coremod.item.permissionscepter.permission.deny=You do not have permission to edit blocks with this tool.
 com.minecolonies.coremod.item.permissionscepter.addblock.success=Added block type '%s'
 com.minecolonies.coremod.item.permissionscepter.addposition.success=Added location '%d %d %d'
 com.minecolonies.coremod.item.permissionscepter.removeblock.success=Removed block '%s'
 com.minecolonies.coremod.item.permissionscepter.removelocation.success=Removed location '%d %d %d'
-com.minecolonies.coremod.gui.townHall.currentHappiness=Overall happiness:
```
Comment (Raycoms) [inline src/main/resources/assets/minecolonies/lang/en_us.lang]: Seems like things happen here you don't want

Comment (xavierh) [reply src/main/resources/assets/minecolonies/lang/en_us.lang]: Different order in 1.10 than 1.11 ?

Comment (Raycoms) [reply src/main/resources/assets/minecolonies/lang/en_us.lang]: It seems like we're losing a few of them.

Comment (Asherslab) [reply src/main/resources/assets/minecolonies/lang/en_us.lang]: hmm?

Comment (Asherslab) [reply src/main/resources/assets/minecolonies/lang/en_us.lang]: it's on line 391, not sure why it moved

Comment (Raycoms) [reply src/main/resources/assets/minecolonies/lang/en_us.lang]: good that I noticed it. There is no en_us in 1.10 its en_US

Comment (Asherslab) [reply src/main/resources/assets/minecolonies/lang/en_us.lang]: so the file name needs to change?

Comment (Raycoms) [reply src/main/resources/assets/minecolonies/lang/en_us.lang]: Yes, and I think the baker things got lost here as well (you can add the baker to the statistic here as well)

Comment (Asherslab) [reply src/main/resources/assets/minecolonies/lang/en_us.lang]: Okay, so change the file name to uppercase US and add Baker stats

**Comment by MinecoloniesSonar** [comment]
SonarQube analysis reported 34 issues
* ![MAJOR][MAJOR] 2 major
* ![MINOR][MINOR] 32 minor

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [WindowTownHall.java#L859](https://github.com/Minecolonies/minecolonies/blob/e87b1b261d20f26e8fc729c220315101035f25ca/src/main/java/com/minecolonies/coremod/client/gui/WindowTownHall.java#L859): Reduce this anonymous class number of lines from 24 to at most 20, or make it a named class. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1188)
1. ![MAJOR][MAJOR] [WindowTownHall.java#L956](https://github.com/Minecolonies/minecolonies/blob/e87b1b261d20f26e8fc729c220315101035f25ca/src/main/java/com/minecolonies/coremod/client/gui/WindowTownHall.java#L956): Reduce this anonymous class number of lines from 40 to at most 20, or make it a named class. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1188)
1. ![MINOR][MINOR] [blockout](https://github.com/Minecolonies/minecolonies/blob/e87b1b261d20f26e8fc729c220315101035f25ca/src/main/java/com/minecolonies/blockout): Add a 'package-info.java' file to document the 'blockout' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [controls](https://github.com/Minecolonies/minecolonies/blob/e87b1b261d20f26e8fc729c220315101035f25ca/src/main/java/com/minecolonies/blockout/controls): Add a 'package-info.java' file to document the 'controls' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [views](https://github.com/Minecolonies/minecolonies/blob/e87b1b261d20f26e8fc729c220315101035f25ca/src/main/java/com/minecolonies/blockout/views): Add a 'package-info.java' file to document the 'views' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [blocks](https://github.com/Minecolonies/minecolonies/blob/e87b1b261d20f26e8fc729c220315101035f25ca/src/main/java/com/minecolonies/coremod/blocks): Add a 'package-info.java' file to document the 'blocks' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [gui](https://github.com/Minecolonies/minecolonies/blob/e87b1b261d20f26e8fc729c220315101035f25ca/src/main/java/com/minecolonies/coremod/client/gui): Add a 'package-info.java' file to document the 'gui' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [WindowTownHall.java#L880](https://github.com/Minecolonies/minecolonies/blob/e87b1b261d20f26e8fc729c220315101035f25ca/src/main/java/com/minecolonies/coremod/client/gui/WindowTownHall.java#L880): Define the locale to be used in this String operation. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1449)
1. ![MINOR][MINOR] [render](https://github.com/Minecolonies/minecolonies/blob/e87b1b261d20f26e8fc729c220315101035f25ca/src/main/java/com/minecolonies/coremod/client/render): Add a 'package-info.java' file to document the 'render' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [colony](https://github.com/Minecolonies/minecolonies/blob/e87b1b261d20f26e8fc729c220315101035f25ca/src/main/java/com/minecolonies/coremod/colony): Add a 'package-info.java' file to document the 'colony' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by MinecoloniesSonar** [comment]
SonarQube analysis reported 34 issues
* ![MAJOR][MAJOR] 2 major
* ![MINOR][MINOR] 32 minor

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [WindowTownHall.java#L859](https://github.com/Minecolonies/minecolonies/blob/b877cd41d55c02cf1e519a84b152fa8a79cd3ce6/src/main/java/com/minecolonies/coremod/client/gui/WindowTownHall.java#L859): Reduce this anonymous class number of lines from 24 to at most 20, or make it a named class. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1188)
1. ![MAJOR][MAJOR] [WindowTownHall.java#L956](https://github.com/Minecolonies/minecolonies/blob/b877cd41d55c02cf1e519a84b152fa8a79cd3ce6/src/main/java/com/minecolonies/coremod/client/gui/WindowTownHall.java#L956): Reduce this anonymous class number of lines from 40 to at most 20, or make it a named class. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1188)
1. ![MINOR][MINOR] [blockout](https://github.com/Minecolonies/minecolonies/blob/b877cd41d55c02cf1e519a84b152fa8a79cd3ce6/src/main/java/com/minecolonies/blockout): Add a 'package-info.java' file to document the 'blockout' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [controls](https://github.com/Minecolonies/minecolonies/blob/b877cd41d55c02cf1e519a84b152fa8a79cd3ce6/src/main/java/com/minecolonies/blockout/controls): Add a 'package-info.java' file to document the 'controls' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [views](https://github.com/Minecolonies/minecolonies/blob/b877cd41d55c02cf1e519a84b152fa8a79cd3ce6/src/main/java/com/minecolonies/blockout/views): Add a 'package-info.java' file to document the 'views' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [blocks](https://github.com/Minecolonies/minecolonies/blob/b877cd41d55c02cf1e519a84b152fa8a79cd3ce6/src/main/java/com/minecolonies/coremod/blocks): Add a 'package-info.java' file to document the 'blocks' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [gui](https://github.com/Minecolonies/minecolonies/blob/b877cd41d55c02cf1e519a84b152fa8a79cd3ce6/src/main/java/com/minecolonies/coremod/client/gui): Add a 'package-info.java' file to document the 'gui' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [WindowTownHall.java#L880](https://github.com/Minecolonies/minecolonies/blob/b877cd41d55c02cf1e519a84b152fa8a79cd3ce6/src/main/java/com/minecolonies/coremod/client/gui/WindowTownHall.java#L880): Define the locale to be used in this String operation. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1449)
1. ![MINOR][MINOR] [render](https://github.com/Minecolonies/minecolonies/blob/b877cd41d55c02cf1e519a84b152fa8a79cd3ce6/src/main/java/com/minecolonies/coremod/client/render): Add a 'package-info.java' file to document the 'render' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [colony](https://github.com/Minecolonies/minecolonies/blob/b877cd41d55c02cf1e519a84b152fa8a79cd3ce6/src/main/java/com/minecolonies/coremod/colony): Add a 'package-info.java' file to document the 'colony' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by MinecoloniesSonar** [comment]
SonarQube analysis reported 34 issues
* ![MAJOR][MAJOR] 2 major
* ![MINOR][MINOR] 32 minor

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [WindowTownHall.java#L859](https://github.com/Minecolonies/minecolonies/blob/6a5545e44f9bf5347194df3a9bcb2dfbb7fa9b13/src/main/java/com/minecolonies/coremod/client/gui/WindowTownHall.java#L859): Reduce this anonymous class number of lines from 24 to at most 20, or make it a named class. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1188)
1. ![MAJOR][MAJOR] [WindowTownHall.java#L956](https://github.com/Minecolonies/minecolonies/blob/6a5545e44f9bf5347194df3a9bcb2dfbb7fa9b13/src/main/java/com/minecolonies/coremod/client/gui/WindowTownHall.java#L956): Reduce this anonymous class number of lines from 40 to at most 20, or make it a named class. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1188)
1. ![MINOR][MINOR] [blockout](https://github.com/Minecolonies/minecolonies/blob/6a5545e44f9bf5347194df3a9bcb2dfbb7fa9b13/src/main/java/com/minecolonies/blockout): Add a 'package-info.java' file to document the 'blockout' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [controls](https://github.com/Minecolonies/minecolonies/blob/6a5545e44f9bf5347194df3a9bcb2dfbb7fa9b13/src/main/java/com/minecolonies/blockout/controls): Add a 'package-info.java' file to document the 'controls' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [views](https://github.com/Minecolonies/minecolonies/blob/6a5545e44f9bf5347194df3a9bcb2dfbb7fa9b13/src/main/java/com/minecolonies/blockout/views): Add a 'package-info.java' file to document the 'views' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [blocks](https://github.com/Minecolonies/minecolonies/blob/6a5545e44f9bf5347194df3a9bcb2dfbb7fa9b13/src/main/java/com/minecolonies/coremod/blocks): Add a 'package-info.java' file to document the 'blocks' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [gui](https://github.com/Minecolonies/minecolonies/blob/6a5545e44f9bf5347194df3a9bcb2dfbb7fa9b13/src/main/java/com/minecolonies/coremod/client/gui): Add a 'package-info.java' file to document the 'gui' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [WindowTownHall.java#L880](https://github.com/Minecolonies/minecolonies/blob/6a5545e44f9bf5347194df3a9bcb2dfbb7fa9b13/src/main/java/com/minecolonies/coremod/client/gui/WindowTownHall.java#L880): Define the locale to be used in this String operation. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1449)
1. ![MINOR][MINOR] [render](https://github.com/Minecolonies/minecolonies/blob/6a5545e44f9bf5347194df3a9bcb2dfbb7fa9b13/src/main/java/com/minecolonies/coremod/client/render): Add a 'package-info.java' file to document the 'render' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [colony](https://github.com/Minecolonies/minecolonies/blob/6a5545e44f9bf5347194df3a9bcb2dfbb7fa9b13/src/main/java/com/minecolonies/coremod/colony): Add a 'package-info.java' file to document the 'colony' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by MinecoloniesSonar** [comment]
SonarQube analysis reported 34 issues
* ![MAJOR][MAJOR] 2 major
* ![MINOR][MINOR] 32 minor

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [WindowTownHall.java#L859](https://github.com/Minecolonies/minecolonies/blob/0119cc485c373d23d1d2ec84745afa48c15497fb/src/main/java/com/minecolonies/coremod/client/gui/WindowTownHall.java#L859): Reduce this anonymous class number of lines from 24 to at most 20, or make it a named class. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1188)
1. ![MAJOR][MAJOR] [WindowTownHall.java#L956](https://github.com/Minecolonies/minecolonies/blob/0119cc485c373d23d1d2ec84745afa48c15497fb/src/main/java/com/minecolonies/coremod/client/gui/WindowTownHall.java#L956): Reduce this anonymous class number of lines from 40 to at most 20, or make it a named class. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1188)
1. ![MINOR][MINOR] [blockout](https://github.com/Minecolonies/minecolonies/blob/0119cc485c373d23d1d2ec84745afa48c15497fb/src/main/java/com/minecolonies/blockout): Add a 'package-info.java' file to document the 'blockout' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [controls](https://github.com/Minecolonies/minecolonies/blob/0119cc485c373d23d1d2ec84745afa48c15497fb/src/main/java/com/minecolonies/blockout/controls): Add a 'package-info.java' file to document the 'controls' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [views](https://github.com/Minecolonies/minecolonies/blob/0119cc485c373d23d1d2ec84745afa48c15497fb/src/main/java/com/minecolonies/blockout/views): Add a 'package-info.java' file to document the 'views' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [blocks](https://github.com/Minecolonies/minecolonies/blob/0119cc485c373d23d1d2ec84745afa48c15497fb/src/main/java/com/minecolonies/coremod/blocks): Add a 'package-info.java' file to document the 'blocks' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [gui](https://github.com/Minecolonies/minecolonies/blob/0119cc485c373d23d1d2ec84745afa48c15497fb/src/main/java/com/minecolonies/coremod/client/gui): Add a 'package-info.java' file to document the 'gui' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [WindowTownHall.java#L880](https://github.com/Minecolonies/minecolonies/blob/0119cc485c373d23d1d2ec84745afa48c15497fb/src/main/java/com/minecolonies/coremod/client/gui/WindowTownHall.java#L880): Define the locale to be used in this String operation. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1449)
1. ![MINOR][MINOR] [render](https://github.com/Minecolonies/minecolonies/blob/0119cc485c373d23d1d2ec84745afa48c15497fb/src/main/java/com/minecolonies/coremod/client/render): Add a 'package-info.java' file to document the 'render' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [colony](https://github.com/Minecolonies/minecolonies/blob/0119cc485c373d23d1d2ec84745afa48c15497fb/src/main/java/com/minecolonies/coremod/colony): Add a 'package-info.java' file to document the 'colony' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by Raycoms** [comment]
:shipit: 

[Approved with PullApprove](https://pullapprove.com/Minecolonies/minecolonies/pull-request/1130/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=Minecolonies/minecolonies)

**Comment by marvin-bitterlich** [comment]
LGTM

[Approved with PullApprove](https://pullapprove.com/Minecolonies/minecolonies/pull-request/1130/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=Minecolonies/minecolonies)

**Pull request merged**